src/hooks/useApiWithProfile.ts

```diff
@@ -31,15 +46,18 @@ export const useApiWithProfile = () => {
 
   const updateAvatar = useRecoilLoadCatch(
     () => async (newAvatar: File) => {
-      await getApiService().uploadAvatar(newAvatar);
+      // TODO: ideally we would use useTypedMutation instead of this but I couldn't get the variables to work w/ mutation -CryptoGraffe
+      const image_data_base64 = await fileToBase64(newAvatar);
```

Comment:
We could switch this and updateBackground back temporarily to the old endpoint until the hasura prod instance is ready

Reply:
I'm not sure that we need to do that since main isn't going to prod

Reply:
quite sure main auto-deploys to prod based on vercel. We'll just do a quick check on the upload mutation functionality once this gets merged.